notifications.bs

```diff
@@ -119,6 +119,12 @@ also available within the web application.
 <a lt="concept notification">notification</a> with an associated
 <a>service worker registration</a>.
 
+<p>A <var>notification</var> is considered to be <dfn>replaceable</dfn> if there
+is a <a lt="concept notification">notification</a> in the <a>list of notifications</a>
+whose <a>tag</a> is not the empty string and equals the <var>notification</var>'s
+tag, and whose <a lt="concept origin">origin</a> is <a>same origin</a> with
```

Comment:
xref the second "tag"

Column width can be 100 these days by the way, though I'll accept this either way.